libs/client/shell/src/lib/activityfeed/activityposts/activityposts.component.scss

```diff
@@ -15,3 +15,9 @@
     font-size: 1.5em;
   }
 }
+
+// #red-badge {
+//   margin-bottom: 40%;
+//   margin-left: 70%;

```

Comment:
@AlexanderChiuy if this is dead code maybe we don't include?

Reply:
Good point